pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CancerRoute.java

```diff
@@ -1,30 +1,17 @@
 package org.broadinstitute.dsm.route;
 
-import java.util.ArrayList;
-import java.util.List;
-
+import lombok.extern.slf4j.Slf4j;
 import org.broadinstitute.dsm.db.Cancer;
-import org.broadinstitute.dsm.statics.UserErrorMessages;
-import org.broadinstitute.lddp.handlers.util.Result;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+
 import spark.Request;
 import spark.Response;
 import spark.Route;
 
+@Slf4j
 public class CancerRoute implements Route {
 
-    private static final Logger logger = LoggerFactory.getLogger(CancerRoute.class);
-
     @Override
     public Object handle(Request request, Response response) throws Exception {
-        List<String> cancerList = new ArrayList<>();
-        try {
-            cancerList = Cancer.getCancers();
-            return cancerList;
-        } catch (Exception e) {
-            logger.error("Cancer list attempt gave an error: ", e);
-            return new Result(500, UserErrorMessages.CONTACT_DEVELOPER);
-        }
+        return Cancer.getCancers();
```

Comment:
I thought this could be simplified quite a bit--if something blows up, let it blow up naturally instead of doing all the wrapping.